src/org/zaproxy/zap/extension/fuzz/httpfuzzer/ui/HttpFuzzerResultsTableModel.java

```diff
@@ -262,6 +263,19 @@ public String getType() {
         public Map<String, Object> getCustomStates() {
             return customStates;
         }
+        /** 
+         * Represents the 'FuzzResultTableEntry' as a string with all values separated by a comma. Is called by
+         * the Export button's actionListener in 'HttpFuzzResultsContentPanel'. 
+         * 
+         * A HttpFuzzerReflectionDetectorStateHighlighter is used to reveal true value of the customState field
+         */
+        public String toString() {

```

Comment:
Got it. I've begun researching the Commons CSV library and will make the changes in the next commit.